src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -150,106 +149,129 @@ Dashboards
 
     <tr>
       <td>
-Alerts      </td>
+Data partitions
```

Comment:
I believe this should say "Log management".  And [this pag](https://docswebsitedevelop-alphabetizetutorialtable.gtsb.io/docs/apis/nerdgraph/examples/nerdgraph-data-partition-rules-tutorial)e should probably have the word "log" in the title since this is all specific to the logging product.